src/helpers/expense.helpers.js

```diff
@@ -0,0 +1,70 @@
+const userService = require('../services/user.service');
+const expenseService = require('../services/expense.service');
+
+const isUserExist = (userId, res) => {
+  if (!userService.getById(userId)) {
+    res.status(400).send('User not found');
```

Comment:
Helpers should know anything about the response and what status/message to send. Just check if user exists